### PR TITLE
Update two_factor_authentication.zh-cn.md

### PR DESCRIPTION
**Pull request by TestOpsCloudchen**
localization

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

localization

### Motivation and Context

for chinese 

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/worst_practices/two_factor_authentication.zh-cn.md [hunk 1]
```diff
@@ -9,18 +9,13 @@ English to Chinese. Do you speak Chinese? Help us to translate
 it by sending us pull requests!
 {{% /notice %}}
```
Comment (Harsha509) [inline docs_source_files/content/worst_practices/two_factor_authentication.zh-cn.md:10]: I think this header should be removed if the translation is done!

**Comment by TestOpsCloudchen** [comment]
> you please sign CLA to get this merged.

acked

File: docs_source_files/content/worst_practices/two_factor_authentication.zh-cn.md (hunk 1 above)
Comment (TestOpsCloudchen) [reply docs_source_files/content/worst_practices/two_factor_authentication.zh-cn.md:10]: i think so

**Comment by TestOpsCloudchen** [comment]
> [CLA assistant check](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=463)
> All committers have signed the CLA.

acked

**Comment by Harsha509** [comment]
Congratulations on your first contribution @TestOpsCloudchen !

**Pull request merged**